test/unit/settings.js

```diff
@@ -95,4 +95,12 @@ describe('settings', function () {
       await cozy.client.settings.deleteClientById('123')
     })
   })
+
+  describe('call the synchronisation route', function () {
+    before(mock.mockAPI('SyncedClient'))
+
+    it('should work', async function () {
+      await cozy.client.settings.updateLastSync()
```

Comment:
It does not test anything but that a `POST` request is sent to `/settings/synchronized`.

---

Ça ne teste pas grand chose, si ce n'est qu'une requête `POST` est envoyée sur une route `/settings/synchronized`.

Reply:
Tout à fait… mais je ne vois que ça a tester.